requirements/huggingface.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
please re-run `make pip-compile` with python3.8 (on mac)

Reply:
actually i thought we move it to 3.10 (its the version from readme